core/src/main/java/io/temporal/samples/hello/HelloDynamic.java

```diff
@@ -54,18 +60,57 @@ public Object execute(EncodedValues args) {
       // Register dynamic signal handler
       Workflow.registerListener(
           (DynamicSignalHandler)
-              (signalName, encodedArgs) -> name = encodedArgs.get(0, String.class));
-
+              (signalName, encodedArgs) -> names.add(encodedArgs.get(0, String.class)));
+
+      // Register dynamic update handler
+      DynamicUpdateHandler updateHandler =
```

Comment:
I think the `DynamicUpdateHandler ` API here is flawed, we should change the API before adding a sample